### PR TITLE
test(ci): show dummy columns test

### PR DESCRIPTION
**Pull request by shinyano**
# New test:
- show dummy data columns after expansion

dummy data expansion
- insert dummy data with new key & old columns: should be able to query
- insert dummy data with columns out of range: should not be able to query

add storage engine
- when wrong params(extra params) are provided for dummy engine, client should fail.

### CODE REVIEW

**Comment by SolomonAnn** [comment]
key的最小值是0还是Long.MIN_VALUE？pr里没有统一，对接层都是Long.MIN_VALUE，但StorageManager.java:98和DefaultMetaManager.java:1257两个地方是0

File: core/src/main/java/cn/edu/tsinghua/iginx/metadata/DefaultMetaManager.java [hunk 1]
```diff
@@ -1252,6 +1252,10 @@ private List<StorageEngineMeta> resolveStorageEngineFromConf() {
       if (hasData) {
         StorageUnitMeta dummyStorageUnit = new StorageUnitMeta(generateDummyStorageUnitId(i), i);
         Pair<ColumnsInterval, KeyInterval> boundary = StorageManager.getBoundaryOfStorage(storage);
+        if (boundary == null) {
```
Comment (SolomonAnn) [inline core/src/main/java/cn/edu/tsinghua/iginx/metadata/DefaultMetaManager.java:1255]: 如果等于null，说明获取范围报错，是否应该打印报错信息后continue？

**Comment by shinyano** [comment]
> key的最小值是0还是Long.MIN_VALUE？pr里没有统一，对接层都是Long.MIN_VALUE，但StorageManager.java:98和DefaultMetaManager.java:1257两个地方是0

统一为long.min-long.max，influxdb特殊处理为min+1-max

File: core/src/main/java/cn/edu/tsinghua/iginx/engine/physical/storage/StorageManager.java [hunk 2]
```diff
@@ -85,6 +85,7 @@ public static Pair<ColumnsInterval, KeyInterval> getBoundaryOfStorage(
       LOGGER.error("load class {} for engine {} failure: {}", driver, engine, e);
```
[the remark this answers is not in the document]
Comment (shinyano) [reply core/src/main/java/cn/edu/tsinghua/iginx/engine/physical/storage/StorageManager.java:85]: 已修改

File: core/src/main/java/cn/edu/tsinghua/iginx/metadata/DefaultMetaManager.java (hunk 1 above)
Comment (shinyano) [reply core/src/main/java/cn/edu/tsinghua/iginx/metadata/DefaultMetaManager.java:1255]: 已修改

**Review by SolomonAnn — APPROVED** [review]

**Pull request merged**